babel.config.js

```diff
@@ -1,3 +1,7 @@
 module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-typescript', ['@babel/preset-react', { runtime: 'automatic' }]],
+  env: {
```

Comment:
what's this change?

Reply:
Wanted to use this presets only on test env